proto/expression.proto

```diff
@@ -142,6 +142,20 @@ enum ExprType {
 	Greatest = 3502;
 	Least = 3503;
 
+	/* Json functions; */
+	JsonExtract = 3601;
+	JsonType = 3602;
+	JsonArray = 3603;
+	JsonObject = 3604;
+	JsonMerge = 3605;
+	JsonValid = 3606;
+	JsonSet = 3607;
+	JsonInsert = 3608;
+	JsonReplace = 3609;
+	JsonRemove = 3610;
+	JsonContains = 3611;
```

Comment:
Would we support [JSON_CONTAINS_PATH](https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#function_json-contains-path) in the future?